contracts/examples/RBACWithAdmin.sol

```diff
@@ -7,14 +7,14 @@ import "../ownership/rbac/RBAC.sol";
  * @title RBACWithAdmin
  * @author Matt Condon (@Shrugs)
  * @dev It's recommended that you define constants in the contract,
- * @dev like ROLE_ADMIN below, to avoid typos.
- * @dev
- * @dev NOTE: RBACWithAdmin is probably too expansive and powerful for your
- * @dev  application; an admin is actually able to change any address to any role
- * @dev  which is a very large API surface. It's recommended that you follow a strategy
- * @dev  of strictly defining the abilities of your roles
- * @dev  and the API-surface of your contract.
- * @dev  This is just an example for example's sake.
+ * like ROLE_ADMIN below, to avoid typos.
+ *
+ * NOTE: RBACWithAdmin is probably too expansive and powerful for your
```

Comment:
sorry, this should be a `@notice` as well!

Reply:
Done.